src/database/index.tsx

```diff
@@ -41,6 +42,9 @@ const scenariosCollectionId = "scenarios";
 const facilitiesCollectionId = "facilities";
 const modelVersionCollectionId = "modelVersions";
 const usersCollectionId = "users";
+// TODO (#521): when the datasource stabilizes, change this to the real collection
```

Comment:
FYI, if anyone is interested in testing using real data, the `reference_facilities` collection has been populated for ~1300 facilities, each with roughly 90 days of daily covid case data.